metrics/sinks/influxdb/influxdb.go

```diff
@@ -33,6 +33,9 @@ type influxdbSink struct {
 	sync.RWMutex
 	c        influxdb_common.InfluxdbConfig
 	dbExists bool
+
+	sync.WaitGroup
```

Comment:
don't embed this, it looks weird calling `sink.Add`

Reply:
Will update `sync.WaitGroup` use a field `wg` instead of embedding.